site/src/pages/WorkspacePage/WorkspaceActions/Buttons.tsx

```diff
@@ -21,19 +20,39 @@ export interface ActionButtonProps {
 	tooltipText?: string;
 }
 
-export const UpdateButton: FC<ActionButtonProps> = ({
+export const UpdateAndStartButton: FC<ActionButtonProps> = ({
```

Comment:
I would probably keep this as one component that takes a prop like `isRunning` or something to change the text, since that seems to be the main difference